src/main/kotlin/com/wire/bots/application/ReminderEventHandler.kt

```diff
@@ -0,0 +1,105 @@
+package com.wire.bots.application
+
+import arrow.core.Either
+import com.wire.bots.domain.event.BotError
+import com.wire.bots.domain.event.Command
+import com.wire.bots.domain.event.EventProcessor
+import com.wire.sdk.WireEventsHandlerSuspending
+import com.wire.sdk.model.WireMessage
+import org.slf4j.LoggerFactory
+
+class ReminderEventHandler(
+    private val eventProcessor: EventProcessor
+) : WireEventsHandlerSuspending() {
+    private val logger = LoggerFactory.getLogger(this::class.java)
+
+    override suspend fun onTextMessageReceived(wireMessage: WireMessage.Text) {
+        logger.info("Received Text Message : $wireMessage")
+        processEvent(
+            MessageEventDTO(
+                type = EventTypeDTO.NEW_TEXT,
+                userId = wireMessage.sender.id.toString(),
+                conversationId = wireMessage.conversationId,
+                text = TextContent(wireMessage.text)
+            )
+        )
+
+        // Sending a Read Receipt for the received message
+        val receipt = WireMessage.Receipt.create(
+            conversationId = wireMessage.conversationId,
+            type = WireMessage.Receipt.Type.READ,
+            messages = listOf(wireMessage.id.toString())
+        )
+        manager.sendMessageSuspending(message = receipt)
+    }
+
+    override suspend fun onAssetMessageReceived(wireMessage: WireMessage.Asset) {
+        logger.info("Received Asset Message: $wireMessage")
+        // Assets are not handled by reminder bot
+    }
+
+    override suspend fun onCompositeMessageReceived(wireMessage: WireMessage.Composite) {
+        logger.info("Received Composite Message: $wireMessage")
+        // Composite messages are not handled by reminder bot
+    }
```

Comment:
As `WireEventsHandlerSuspending` is an abstract class and there is already a base implementation of both methods above, can we just remove them from here?